dev/utils.ts

```diff
@@ -13,16 +13,17 @@ export async function runOrExit(
   cwd?: string,
   env: Record<string, string> = {},
 ) {
+  // console.error("shell: ", cmd.join(" "), { env, cwd });
   const p = new Deno.Command(cmd[0], {
     args: cmd.slice(1),
     cwd,
-    stdout: "piped",
-    stderr: "piped",
+    // stdout: "piped",
+    // stderr: "piped",
     env: { ...Deno.env.toObject(), ...env },
   }).spawn();
 
-  await p.stdout.pipeTo(Deno.stdout.writable, { preventClose: true });
-  await p.stderr.pipeTo(Deno.stderr.writable, { preventClose: true });
+  // await p.stdout.pipeTo(Deno.stdout.writable, { preventClose: true });
```

Comment:
a leftover?

Reply:
@Yohe-Am was this a fix or..?

Reply:
This was the culprit for the `test-docker` issues or part of it anyways. `dev/env.ts` calls `deno cache` (another problematic API?) with thousands of paths in it's `argv`. I'm not sure what it is about pipes that makes it hang but it works well when it's talking directly to the `tty`.

Reply:
what about `(void) p.stdout ..` and then await the status at the end?

Reply:
Hmm, I was actually a bit confused as to why it was piped in the first place. It only seems to be streaming it to stdout and nowhere else. In that case, shouldn't just let it inherit the std files?

Reply:
@Yohe-Am I do no recall, maybe this is not needed anymore but I recall something required that to the log when they initially switched to that new api